controllers/mlmd_test.go

```diff
@@ -18,14 +18,15 @@ limitations under the License.
 package controllers
 
 import (
+	v1 "github.com/openshift/api/route/v1"
 	"testing"
 
 	dspav1alpha1 "github.com/opendatahub-io/data-science-pipelines-operator/api/v1alpha1"
 	"github.com/stretchr/testify/assert"
 	appsv1 "k8s.io/api/apps/v1"
 )
 
-func TestDeployMLMD(t *testing.T) {
+func TestDeployMLMDV1(t *testing.T) {
```

Comment:
we have an upcoming item to make v2 default, so perhaps now that we have versioned tests we should also update the DSPA spec used here to explicitly specify DSPVersion: "v1"

Reply:
Done.